tests/ocs-monkey/test_ocs_monkey.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import shlex
+import subprocess
+import os
+import time
+
+from ocs_ci.framework.testlib import ignore_leftovers
+from ocs_ci.utility.utils import clone_repo, run_cmd
+from ocs_ci.ocs import constants
+
+log = logging.getLogger(__name__)
+
+
+@ignore_leftovers
```

Comment:
@hnallurv @PrasadDesala should this test be part of `workloads` tier?

Reply:
IMO it should be in a new tier level.
As this is not a real workload we can try to integrate it with the other workloads that are running for a long time.
I'm thinking mostly about the BDI workload that can run for hours and with the OCS monkey in place, we are really taking the test to the next level

Reply:
yes. I have discussed this with @jilju and he brought up the idea of having a dedicated Jenkins job specifically for ocs-monkey test